build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_cpu_llvm_task.json

```diff
@@ -14,7 +14,9 @@
     "--input=2x1280xf16=@inference_input.2.bin",
     "--input=1xf16=@inference_input.3.bin",
     "--expected_output=1x4x128x128xf16=@inference_output.0.bin",
-    "--expected_f16_threshold=0.8f"
+    "--expected_f16_threshold=0.8f",
+    // TODO(#17467): Remove the workaround once we have better support for attention op codegen.
+    "iree-llvmcpu-fail-on-large-vector=false"
```

Comment:
This should be added to `iree_compile_flags` (up a few lines), not `iree_run_module_flags`. Also missing `--` from the flag

Reply:
Thanks thanks.. I don't know what I was doing. fixed!